pkg/events/logger_handler/logger_handler.go

```diff
@@ -0,0 +1,101 @@
+package logger_handler
+
+import (
+	k8sV1 "k8s.io/api/core/v1"
+	"k8s.io/klog"
+
+	"github.com/submariner-io/admiral/pkg/log"
+	submV1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
+	"github.com/submariner-io/submariner/pkg/events"
+)
+
+type LoggerHandler struct {
```

Comment:
```suggestion
type Handler struct {
```

Also rename package to `logger` so  references become `logger.Handler`. Also rename file to _handler.go_.

Reply:
oh, this is part of this commit, yikes, I didn't handle this part. I'll do